geniza/corpus/models.py

```diff
@@ -622,6 +607,36 @@ def sources(self):
         """All unique sources attached to footnotes on this document."""
         return Source.objects.filter(footnote__document=self).distinct()
 
+    def attribution(self):
```

Comment:
wow, thanks for generalizing this.

Now that I see it on the document — so much clearer than wherever it gets displayed in mirador — I think we need to figure out how to clarify what the Cambridge rights statement applies to (especially since we're not! displaying their metadata). Maybe display per shelfmark? Or group of shelfmark if they're the same, but sometimes there is small variation (IIRC some of the Cambridge attribution text had links but some did not; otherwise they were the same.)  

I don't think we need to do anything about this right away, though! Let's get feedback and go-ahead on the approach first.